std/algebra/weierstrass/params.go

```diff
@@ -25,11 +27,525 @@ type CurveParams struct {
+	g2To254y, _ := new(big.Int).SetString("1dfeda5c16e651fbac7b5ad608b96cf5e01eaec17a02182f96ccf5252e76373", 16)
+	g2To255x, _ := new(big.Int).SetString("b23790a42be63e1b251ad6c94fdef07271ec0aada31db6c3e8bd32043f8be384", 16)
+	g2To255y, _ := new(big.Int).SetString("fc6b694919d55edbe8d50f88aa81f94517f004f4149ecb58d10a473deb19880e", 16)
 	return CurveParams{
-		A:  big.NewInt(0),
-		B:  big.NewInt(7),
-		Gx: gx,
-		Gy: gy,
+		A:   big.NewInt(0),
+		B:   big.NewInt(7),
+		Gx:  gx,
+		Gy:  gy,
+		Gmx: [256]*big.Int{g3x, g5x, g7x, g2To3x, g2To4x, g2To5x, g2To6x, g2To7x, g2To8x, g2To9x, g2To10x, g2To11x, g2To12x, g2To13x, g2To14x, g2To15x, g2To16x, g2To17x, g2To18x, g2To19x, g2To20x, g2To21x, g2To22x, g2To23x, g2To24x, g2To25x, g2To26x, g2To27x, g2To28x, g2To29x, g2To30x, g2To31x, g2To32x, g2To33x, g2To34x, g2To35x, g2To36x, g2To37x, g2To38x, g2To39x, g2To40x, g2To41x, g2To42x, g2To43x, g2To44x, g2To45x, g2To46x, g2To47x, g2To48x, g2To49x, g2To50x, g2To51x, g2To52x, g2To53x, g2To54x, g2To55x, g2To56x, g2To57x, g2To58x, g2To59x, g2To60x, g2To61x, g2To62x, g2To63x, g2To64x, g2To65x, g2To66x, g2To67x, g2To68x, g2To69x, g2To70x, g2To71x, g2To72x, g2To73x, g2To74x, g2To75x, g2To76x, g2To77x, g2To78x, g2To79x, g2To80x, g2To81x, g2To82x, g2To83x, g2To84x, g2To85x, g2To86x, g2To87x, g2To88x, g2To89x, g2To90x, g2To91x, g2To92x, g2To93x, g2To94x, g2To95x, g2To96x, g2To97x, g2To98x, g2To99x, g2To100x, g2To101x, g2To102x, g2To103x, g2To104x, g2To105x, g2To106x, g2To107x, g2To108x, g2To109x, g2To110x, g2To111x, g2To112x, g2To113x, g2To114x, g2To115x, g2To116x, g2To117x, g2To118x, g2To119x, g2To120x, g2To121x, g2To122x, g2To123x, g2To124x, g2To125x, g2To126x, g2To127x, g2To128x, g2To129x, g2To130x, g2To131x, g2To132x, g2To133x, g2To134x, g2To135x, g2To136x, g2To137x, g2To138x, g2To139x, g2To140x, g2To141x, g2To142x, g2To143x, g2To144x, g2To145x, g2To146x, g2To147x, g2To148x, g2To149x, g2To150x, g2To151x, g2To152x, g2To153x, g2To154x, g2To155x, g2To156x, g2To157x, g2To158x, g2To159x, g2To160x, g2To161x, g2To162x, g2To163x, g2To164x, g2To165x, g2To166x, g2To167x, g2To168x, g2To169x, g2To170x, g2To171x, g2To172x, g2To173x, g2To174x, g2To175x, g2To176x, g2To177x, g2To178x, g2To179x, g2To180x, g2To181x, g2To182x, g2To183x, g2To184x, g2To185x, g2To186x, g2To187x, g2To188x, g2To189x, g2To190x, g2To191x, g2To192x, g2To193x, g2To194x, g2To195x, g2To196x, g2To197x, g2To198x, g2To199x, g2To200x, g2To201x, g2To202x, g2To203x, g2To204x, g2To205x, g2To206x, g2To207x, g2To208x, g2To209x, g2To210x, g2To211x, g2To212x, g2To213x, g2To214x, g2To215x, g2To216x, g2To217x, g2To218x, g2To219x, g2To220x, g2To221x, g2To222x, g2To223x, g2To224x, g2To225x, g2To226x, g2To227x, g2To228x, g2To229x, g2To230x, g2To231x, g2To232x, g2To233x, g2To234x, g2To235x, g2To236x, g2To237x, g2To238x, g2To239x, g2To240x, g2To241x, g2To242x, g2To243x, g2To244x, g2To245x, g2To246x, g2To247x, g2To248x, g2To249x, g2To250x, g2To251x, g2To252x, g2To253x, g2To254x, g2To255x},
```

Comment:
Even if lazily computing the points is too time-consuming, then I would still use a table structure instead of a list of variables. It is just a little less error-prone.

Reply:
makes sense